x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/aws_credentials_form/get_aws_credentials_form_options.tsx

```diff
@@ -140,8 +145,12 @@ export const getAwsCredentialsFormOptions = (): AwsOptions => ({
     }),
     info: DirectAccessKeysDescription,
     fields: {
-      access_key_id: { label: AWS_FIELD_LABEL.access_key_id },
-      secret_access_key: { label: AWS_FIELD_LABEL.secret_access_key, type: 'password' },
+      access_key_id: { label: AWS_FIELD_LABEL.access_key_id, dataTestId: 'directAccessKeyId' },
```

Comment:
I'd prefix cloud specific test ids with a cloud prefix, eg. `awsDirectAccessKeyId`. While direct-access key is specific to AWS , we already have examples where it's hard to tell which cloud the value belongs to, eg. for `manual`

Reply:
and you already have it for azure, consistency is another reason to add this to AWS as well